extra-dev/packages/coq-mi-cho-coq/coq-mi-cho-coq.dev/opam

```diff
@@ -0,0 +1,37 @@
+opam-version: "2.0"
+synopsis: "A specification of Michelson in Coq to prove properties about smart contracts in Tezos"
+maintainer: "raphael.cauderlier@nomadic-labs.com"
+authors: [ "Raphaël Cauderlier" "Bruno Bernardo" "Julien Tesson" "Arvid Jakobsson" ]
+
+homepage: "https://gitlab.com/nomadic-labs/mi-cho-coq/"
+dev-repo: "git+https://gitlab.com/nomadic-labs/mi-cho-coq/"
+bug-reports: "https://gitlab.com/nomadic-labs/mi-cho-coq/issues"
+license: "MIT"
+
+build: [
+  ["./configure"]
+  [make "-j%{jobs}%"]
+]
+install: [
+  make "install"
+]
+depends: [
```

Comment:
```suggestion
depends: [
  "ocaml"
```
Since we're using Ott here (and are also targeting extraction), I think OCaml should be an explicit dependency, but this could be a matter of taste.